tests/AliasesTest.php

```diff
@@ -10,6 +10,40 @@
 
 final class AliasesTest extends TestCase
 {
+    public function dataBase(): array
+    {
+        return [
+            ['', ['@url' => ''], '@url'],
+            ['/', ['@url' => ''], '@url/'],
+            ['/about', ['@url' => ''], '@url/about'],
+            ['/', ['@url' => '/'], '@url'],
+            ['/', ['@url' => '/'], '@url/'],
+            ['/about', ['@url' => '/'], '@url/about'],
+            ['test', ['@url' => 'test'], '@url'],
+            ['test/', ['@url' => 'test'], '@url/'],
+            ['test/about', ['@url' => 'test'], '@url/about'],
+            ['test/', ['@url' => 'test/'], '@url'],
+            ['test/', ['@url' => 'test/'], '@url/'],
+            ['test/about', ['@url' => 'test/'], '@url/about'],
+            ['/test', ['@url' => '/test'], '@url'],
+            ['/test/', ['@url' => '/test'], '@url/'],
+            ['/test/about', ['@url' => '/test'], '@url/about'],
+            ['/test/', ['@url' => '/test/'], '@url'],
```

Comment:
even if internally we do not trim slashes, this one should return `/test` and not `/test/`.

```php
$a = $aliases->get('@url') . '/subpath'; 
// expect $a to be "/test/subpath" and not "/test//subpath"
```

Reply:
Hm. But if alias will be `/` then:

```
$a = $aliases->get('@url') . '/subpath'; 
// "//subpath"
```

May be don't change behavior and close this PR? =)

Reply:
> Hm. But if alias will be `/` then:
> 
> ```
> $a = $aliases->get('@url') . '/subpath'; 
> // "//subpath"
> ```
> 
> May be don't change behavior and close this PR? =) Now I like this choice.